src/viam/app/data_client.py

```diff
@@ -120,10 +120,15 @@ def __init__(self, channel: Channel, metadata: Mapping[str, str]):
         self._metadata = metadata
         self._data_client = DataServiceStub(channel)
         self._data_sync_client = DataSyncServiceStub(channel)
+        self._channel = channel
 
     _data_client: DataServiceStub
     _data_sync_client: DataSyncServiceStub
     _metadata: Mapping[str, str]
+    _channel: Channel
+
+    def close(self) -> None:
+        self._channel.close()
```

Comment:
Is this really needed?

Reply:
It came in handy when I was testing and needed to connect to mock servers and so didnt use ViamClient to create an AppClient but instead just manually instantiated an AppClient directly. I kept getting those warnings that a channel was still open and figured I'd add a method to close for others who might also make an appclient directly - also to avoid making people access the private _channel. Should i take it out?

Reply:
Oh nice ya that's fine you can keep it in